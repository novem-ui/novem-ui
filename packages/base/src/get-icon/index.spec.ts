import getIcon from '.'

describe('get-icon', () => {
  test('should return error icon', () => {
    const icon = getIcon({ error: true })

    expect(icon).toMatchSnapshot()
  })

  test('should return valid icon', () => {
    const icon = getIcon({ valid: true })

    expect(icon).toMatchSnapshot()
  })

  test('should return null', () => {
    const icon = getIcon({})

    expect(icon).toBeNull()
  })
})
