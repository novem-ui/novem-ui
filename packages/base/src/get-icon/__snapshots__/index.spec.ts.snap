// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get-icon should return error icon 1`] = `"\\"data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M16 0H0V16H16V0Z' fill='white' fill-opacity='0.01'/%3E%3Cpath d='M4.66699 4.66663L11.3337 11.3333' stroke='white' stroke-width='1.33333' stroke-linecap='round' stroke-linejoin='round'/%3E%3Cpath d='M4.66699 11.3333L11.3337 4.66663' stroke='white' stroke-width='1.33333' stroke-linecap='round' stroke-linejoin='round'/%3E%3C/svg%3E%0A\\""`;

exports[`get-icon should return valid icon 1`] = `"\\"data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M16 0H0V16H16V0Z' fill='white' fill-opacity='0.01'/%3E%3Cpath d='M3.33301 7.99996L6.66634 11.3333L13.333 4.66663' stroke='white' stroke-width='1.33333' stroke-linecap='round' stroke-linejoin='round'/%3E%3C/svg%3E%0A\\""`;
